pkg/sql/tests/rsg_test.go

```diff
@@ -200,6 +200,14 @@ func (db *verifyFormatDB) execWithResettableTimeout(
 					return &nonCrasher{sql: sql, err: err}
 				}
 				return nil
+			case <-ctx.Done():
+				// Sanity: When our context gets cancelled and the test is
```

Comment:
Are context cancellation and test timeout correlated? The current phrasing feels a bit ambiguous. Maybe something like "ctx is cancelled when the test is about to timeout. Log out whatever statement we're waiting on for debugging purposes. Sometimes queries won't respect cancellation due to lib/pq being weird". 

In the case that we do log out a query, are we trying to "save" the test from timing out or just leaving some breadcrumbs?

Reply:
Updated the comment				
                                 // Sanity: The context is cancelled when the test is about to
				// timeout. We will log whatever statement we're waiting on for
				// debugging purposes. Sometimes queries won't respect
				// cancellation due to lib/pq limitations.

Yes,, the goal here is to get some bread crumbs out; otherwise, we have to look at server stacks to figure out what is executing. This gives us a better understanding modulo some noise in the successful cases.